crates/modelardb_compression/src/models/swing.rs

```diff
@@ -56,6 +56,10 @@ pub struct Swing {
     /// Intercept for the linear function specifying the lower bound for the
     /// current model.
     lower_bound_intercept: f64,
+    /// Computation of the slope to minimize MSE for the current model 
+    /// and holds the numenator and denominator of Equation 6 
```

Comment:
I am not sure this sentence makes sense. Maybe `Computation of the slope to minimize MSE for the current model. Specifically, it is the numerator and denominator...` or something similar instead.

Reply:
Fixed.